hourly-tips/src/main/java/org/apache/flink/training/exercises/hourlytips/HourlyTipsExercise.java

```diff
@@ -68,26 +73,94 @@ public static void main(String[] args) throws Exception {
      * @throws Exception which occurs during job execution.
      */
     public JobExecutionResult execute() throws Exception {
+        return option2();
+    }
 
+    private JobExecutionResult option1() throws Exception {
```

Comment:
would be great if Method has meaningful name, `option1()` doesn't say much

Reply:
well, that is what it is, one option how to do what was requested, and there is option 2 which gives same result, different implementation

Reply:
ok